From a28df3be5df5655501ac5afa351162deddfd031d Mon Sep 17 00:00:00 2001
From: "Marius B. Kotsbak" <marius@kotsbak.com>
Date: Mon, 24 Dec 2012 11:17:40 +0100
Subject: [PATCH] qmi-codegen: clean up compiled python files

These caused problems for deb packaging, as their content depends on the Python version used.
---
 build-aux/qmi-codegen/Makefile.am |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/build-aux/qmi-codegen/Makefile.am b/build-aux/qmi-codegen/Makefile.am
index aa2a14a..95ae999 100644
--- a/build-aux/qmi-codegen/Makefile.am
+++ b/build-aux/qmi-codegen/Makefile.am
@@ -16,3 +16,5 @@ EXTRA_DIST = \
 	VariableString.py \
 	utils.py \
 	qmi-codegen
+
+CLEANFILES = *.pyc
-- 
1.7.10.4

